src/V3File.h

```diff
@@ -61,8 +61,8 @@ class V3File final {
     }
 
     // Dependencies
-    static void addSrcDepend(const string& filename);
-    static void addTgtDepend(const string& filename);
+    static void addSrcDepend(const string& filename) VL_MT_SAFE;
+    static void addTgtDepend(const string& filename) VL_MT_SAFE;
```

Comment:
I don't see a lock in this routine. Also need to be adding attributes to the definition.